docs/topics/RPC.md

```diff
@@ -240,7 +240,7 @@ Used to authenticate an existing client with your app.
 | description | string | application description |
 | icon | string | hash of the icon |
 | id | snowflake | application client id |
-| rpc_origins | array | array of strings - |
+| rpc_origins | array of strings | array of rpc origin url strings |
```

Comment:
"array of rpc origin urls", perhaps? We already specified these are strings.

Reply:
Well perhaps you could make that change there too? It just seems odd to me to specify that they're URL strings, when URLs are *always* strings.

Reply:
Alright